### PR TITLE
Re-order wide bytecode for Valuetype bytecodes

### PR DESCRIPTION
**Pull request by tajila**
Re-order wide bytecode for Valuetype bytecodes

This change shifts internal bytecodes 203-216 up by two, and
moves the Value Types bytecodes to where they belong.

These commit is based on Ali's changes in
https://github.com/tajila/openj9/pull/6/commits.

Signed-off-by: tajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@gacholio can you please review this

**Comment by gacholio** [comment]
bcnames.c also needs to be updated. I'm also not sure we want to ifdef the new bytecodes - the JIT may want them available all the time.

Much as I hate to say it, we probably want to version the bytecode changes for DDR, so it can continue to interpret the old wide bytecode values.

**Comment by tajila** [comment]
@gacholio The PR is ready for another look

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ByteCodeVersion.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+package com.ibm.j9ddr.vm29.j9;
+
+import java.lang.reflect.Method;
+
+import com.ibm.j9ddr.CorruptDataException;
+import com.ibm.j9ddr.vm29.pointer.helper.J9RASHelper;
+import com.ibm.j9ddr.vm29.types.UDATA;
+import com.ibm.j9ddr.vm29.pointer.generated.J9JavaVMPointer;
+
+public class ByteCodeVersion {
```
Comment (gacholio) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ByteCodeVersion.java]: I don't think this is what you want - you want an AlgorithmVersion, which will not take up any space in the VM.

**Comment by tajila** [comment]
@gacholio I've addressed your comment

File: runtime/util/vrfytbl.c [hunk 2]
```diff
@@ -226,36 +226,30 @@ const U_16 J9JavaBytecodeVerificationTable[] = {
...
-		0x1402 /* JBfloadw - 205 */,
-		0x1403 /* JBdloadw - 206 */,
-		0x1407 /* JBaloadw - 207 */,
-		0x1501 /* JBistorew - 208 */,
-		0x1504 /* JBlstorew - 209 */,
-		0x1502 /* JBfstorew - 210 */,
-		0x1503 /* JBdstorew - 211 */,
-		0x1507 /* JBastorew - 212 */,
+		0x1200 /* JBdefaultvalue = 203 */ ,
+		0x1000 /* JBwithfield = 204 */ ,
```
Comment (gacholio) [inline runtime/util/vrfytbl.c]: Extra blank before comma on the previous two lines.

Comment (gacholio) [reply runtime/util/vrfytbl.c]: jenkins test sanity xlinux jdk11

**Comment by gacholio** [comment]
jenkins test sanity xlinux jdk11

**Comment by gacholio** [comment]
jenkins test extended zlinux jdk11

**Comment by gacholio** [comment]
CI build is failing for known reasons unrelated to this PR.

**Comment by sehirst** [comment]
This change was missing the matching JIT change which caused a substantial regression. @cathyzhyi is fixing this in #4344.

Changes to bytecode enums/tables in the VM cannot be made without first consulting the JIT team.

**Pull request merged**